### PR TITLE
Added captcha in singUp form

### PR DESCRIPTION
**Pull request by satcasm**
Fixes: #642 
Screenshot Reference:
image

### CODE REVIEW

File: routes/user.router.js [hunk 1]
```diff
@@ -141,6 +143,27 @@ router.post(
 			password,
 			confirmPassword,
 		} = req.body;
+		if (!req.body['g-recaptcha-response'])
+			return res.json({ success: false, msg: 'Please select captcha' });
+
+		// Secret key
+		const secretKey = '6LecnfAaAAAAAC2AOAYwA2xyHLbiuPqkdJHDHwYX';
```
Comment (Pranav016) [inline routes/user.router.js:150]: shouldn't this secret key stored as an env variable ?

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Lint the code before committing

File: routes/user.router.js (hunk 1 above)
Comment (satcasm) [reply routes/user.router.js:150]: I thought you would be generating a new key as I wasn't sure which domains to give access to, that's why I didn't make it an env variable. 
Kindly go through this: https://www.google.com/recaptcha/admin/create.
Also, select this option:
image
Apologies for inconvenience.

Comment (satcasm) [reply routes/user.router.js:150]: Or I can add word-nook.herokuapp.com/ as a domain and make the secret key as env, whichever works for you.
Let me know.

**Comment by satcasm** [comment]
@Pranav016 any update on this?

File: routes/user.router.js (hunk 1 above)
Comment (Pranav016) [reply routes/user.router.js:150]: Sorry for the late reply, yes you can use ALPHAVIO's account to set the domain and make the secret key as an env. You can get ALPHAVIO's account credentials from me on discord

**Comment by Pranav016** [comment]
> > @Pranav016 any update on this?

> Sorry for the late reply, yes you can use ALPHAVIO's account to set the domain and make the secret key as an env. You can get ALPHAVIO's account credentials from me on discord

yes

**Comment by Pranav016** [comment]
Make the secret as an env variable and resolve conflicts to get it merged

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**